src/Utopia/Messaging/Helpers/JWT.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+namespace Utopia\Messaging\Helpers;
+
+class JWT
```

Comment:
Why can't we use a JWT library instead of implementing it ourselves?

Reply:
A few reasons why doing it ourselves is better in my opinion:
- Minimizes dependencies
- Doesn't add maintenance overhead since JWT encoding is pretty much set-and-forget
- We have the option of publishing our own library in the future